queries.sql

```diff
@@ -8,3 +8,65 @@ SELECT (name, escape_attempts) FROM animals WHERE weight_kg > 10.50;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.40 AND 17.30;
+
+BEGIN;
+UPDATE animals
+SET species = NULL;
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon%';
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+SELECT * FROM animals;
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals;
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals
+WHERE birthdate > '2022-01-01';
+SAVEPOINT SP1;
+UPDATE animals
+SET weight_kg = weight_kg * -1;
+SELECT * FROM animals;
+ROLLBACK TO SP1;
+SELECT * FROM animals;
+UPDATE animals
+SET weight_kg = weight_kg * -1
+WHERE weight_kg < 0;
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN;
+SELECT COUNT(*) FROM animals;
+SELECT 
+   COUNT(escape_attempts) 
+FROM 
+   animals
+WHERE escape_attempts = '0';
+SELECT AVG(weight_kg) FROM animals;
+SELECT neutered, COUNT(*)
+FROM animals
+WHERE escape_attempts > 0
+GROUP BY neutered;
+SELECT neutered,
+       MIN(weight_kg),
+       MAX(weight_kg)
+FROM animals
+GROUP BY neutered;
```

Comment:
👍 Great work on the SQL query! It's well-structured, and you've provided useful insights. However, to meet the project requirements, you should make the following adjustments:

- [ ] Your SQL query currently groups data from the "animals" table by the "neutered" column, calculating the minimum (MIN) and maximum (MAX) values of the "weight_kg" column for each group. This approach offers insights into the range of weights for neutered and non-neutered animals. However, as per the project requirement, your query should answer the question: "What is the minimum and maximum weight of each type of animal?" To meet this requirement, please update your query to calculate the minimum and maximum weight for each distinct animal type within the dataset, providing a comprehensive summary of the weight range for different animal types.
- [ ] If needed, kindly update the screenshot in the pull request (PR) description. Your cooperation is greatly appreciated.

Your dedication to aligning your work with the project's needs is highly appreciated. Keep up the good work! 👏📊🐕